src/usr/local/www/system_advanced_notifications.php

```diff
@@ -230,93 +230,92 @@
 
 $section->addInput(new Form_Input(
 	'smtpipaddress',
-	'E-Mail server',
+	'SMTP server',
 	'text',
 	$pconfig['smtpipaddress']
-))->setHelp('This is the FQDN or IP address of the SMTP E-Mail server to '.
+))->setHelp('The FQDN or IP address of the SMTP mail server to '.
 	'which notifications will be sent.');
 
 $section->addInput(new Form_Input(
 	'smtpport',
-	'SMTP Port of E-Mail server',
+	'SMTP port',
 	'number',
 	$pconfig['smtpport']
-))->setHelp('This is the port of the SMTP E-Mail server, typically 25, 587 '.
+))->setHelp('Port that the SMTP server accepts connections on. Typically 25, 587 '.
 	'(submission) or 465 (smtps).');
 
-$section->addInput(new Form_Input(
-	'smtptimeout',
-	'Connection timeout to E-Mail server',
-	'number',
-	$pconfig['smtptimeout']
-))->setHelp('This is how many seconds it will wait for the SMTP server to connect. Default is 20s.');
-
-$group = new Form_Group('Secure SMTP Connection');
-$group->add(new Form_Checkbox(
+$section->addInput(new Form_Checkbox(
 	'smtpssl',
-	'Enable SSL/TLS',
+	'SSL/TLS encryption',
 	'Enable SMTP over SSL/TLS',
-	isset($pconfig['smtpssl'])
+	$pconfig['smtpssl']
 ));
 
-$section->add($group);
+$section->addInput(new Form_Input(
+	'smtptimeout',
+	'Connection timeout',
+	'number',
+	$pconfig['smtptimeout']
+))->setHelp('Number of seconds to wait for the SMTP connection to succeed. Default is 20.');
 
 $section->addInput(new Form_Input(
 	'smtpfromaddress',
-	'From e-mail address',
+	'From email address',
 	'text',
 	$pconfig['smtpfromaddress']
-))->setHelp('This is the e-mail address that will appear in the from field.');
+))->setHelp('The email address that will appear in the From field.');
 
 $section->addInput(new Form_Input(
 	'smtpnotifyemailaddress',
-	'Notification E-Mail address',
+	'To email address',
 	'text',
 	$pconfig['smtpnotifyemailaddress']
-))->setHelp('Enter the e-mail address to send email notifications to.');
+))->setHelp('The email address to send notifications to. Multiple addresses may be '.
```

Comment:
That first sentence is a bit awkward with the repeated 'to' there (it was already awkward in the original line). Maybe this instead:
> The email address to which notifications will be sent.